include/curl/curl.h

```diff
@@ -3063,7 +3079,7 @@ struct curl_version_info_data {
   const char *version;      /* LIBCURL_VERSION */
   unsigned int version_num; /* LIBCURL_VERSION_NUM */
   const char *host;         /* OS/host/cpu/machine when configured */
-  int features;             /* bitmask, see defines below */
+  long features;             /* bitmask, see defines below */
```

Comment:
ack, will look into ``feature_names``

Reply:
I did that (I thnik) but get a warning about (1<<31) being too big, though things work ok. I guess I'm wondering if it's worthwhile have ECH as a "feature" like that anyway.

Reply:
Since that (1<<31) seems to make many of the CI builds unhappy, and since DoH and HTTPS RR support aren't listed as "features," I'll take that out for now. Can be added back if/when I figure out how to do it right:-)